content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Service Accounts
+description: >
+  Learn about the Kubernetes ServiceAccount object.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page introduces the ServiceAccount object in Kubernetes, providing
+information about how service accounts work, use cases, limitations,
+alternatives, and links to resources for additional guidance.
+
+<!-- body -->
+
+## What are service accounts?
+
+A service account is a type of non-human account that, in Kubernetes, provides
+an identity for the processes that run in Pods. Pods can use service accounts
+to make calls to the Kubernetes API server. Service accounts exist as
+ServiceAccount objects in the API server. Service accounts have the following
+properties:
+
+*  **Namespaced:** Each service account is bound to a Kubernetes
+   {{<glossary_tooltip text="namespace" term_id="namespace">}}. Every namespace
+   gets a `default` ServiceAccount upon creation.
+
+*  **Lightweight:** Service accounts exist in the cluster and use the structure
+   defined in the Kubernetes API. You can quickly create service accounts to
+   enable specific tasks.
+
+*  **Portable:** A configuration bundle for a complex containerized workload
+   might include service account definitions for the system's components. The
+   lightweight nature of service accounts and the namespaced identities make
+   the configurations portable.
+
+Service accounts are different from user accounts, which are authenticated
+human users in the cluster. User accounts don't exist in the Kubernetes API
+server; instead, the API server treats user identities as opaque data. You can
+authenticate as a user account using multiple methods.
+
+{{< table caption="Comparison between service accounts and users" >}}
+
+| Description | ServiceAccount | User or group |
+| --- | --- | --- |
+| Location | Kubernetes API (ServiceAccount object) | External |
+| Scope | Namespace | Cluster |
+| Access control | Kubernetes RBAC or other [authorization mechanisms](/docs/reference/access-authn-authz/authorization/#authorization-modules) | Kubernetes RBAC or other identity and access management mechanisms |
+| Intended use | Pods | People |
+
+{{< /table >}}
+
+### Default service accounts
+
+When you create a cluster, Kubernetes automatically creates a ServiceAccount
+object named `default` for every namespace in your cluster. The `default`
+service accounts in each namespace get no permissions other than the
```

Comment:
they can be given other permissions, but get none by default:

```suggestion
service accounts in each namespace get no permissions by default other than the
```

Reply:
Done